test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,70 @@
+# Kubemark Pre-existing Provider Guide
+
+**Kubemark Master**
+- A Kubernetes master running in a VM that will be used as the kubemark master
+
+**Kubernetes Cluster**
+- A real Kubernetes Cluster that has master and minions. The hollow-node pods
+  are run in this cluster, but appear as nodes to the Kubemark Master
```

Comment:
Wouldn't this be referred to as a cluster? Since it's likely you'll be running many hollow node pods, I'd expect that to require many real nodes. Is the purpose of your comment to make the syntax match the definition above it?